eclass/llvm-utils.eclass

```diff
@@ -150,4 +150,19 @@ llvm_prepend_path() {
 	export PATH=${new_path[*]}
 }
 
+# @FUNCTION: llvm_has_musl_libc
+# @DESCRIPTION:
+# Determine whether the given LLVM project should be built with musl
+# support. That should be the case under the following ciricumstances:
+#
+# - CTARGET is a musl environment.
+# - The host profile has the `elibc_musl` flag.
+llvm_has_musl_libc() {
+	if [[ "${CTARGET}" == *-musl* ]]; then
```

Comment:
Nit: I'd probably make it `*-*-*-musl*` to avoid accidentally matching vendor field. @thesamesam ?

Reply:
Good idea. To be fair, the whole `crossdev` script and some other ebuilds have these exact checks (`*-[some_libc]*`), but I'm happy to file PRs fixing these as well.

Reply:
WFM.

Reply:
Done